reactDashboard/src/components/GeoHeatMap/GeoHeatMap.css

```diff
@@ -41,17 +43,21 @@ div.leaflet-tooltip p {
 }
 
 div.leaflet-tooltip-left::before {
-  border-left-color: #000000;
+  border-left-color: #ac3232;
```

Comment:
changed to red - I'm not sure what this is for

Reply:
Took me a while to find what element this affected. It's the little triangles on the region tooltips. Agreed that these colour changes are unnecessary. If anything, the triangles should match those visible on the overview geomap popups - either both white or both black.

Reply:
I am not sure why these were changed either. Changed it all to black.